UWP/GalaxyExplorer/App.xaml.cs

```diff
@@ -125,6 +126,58 @@ private void InitializeUnity(string args)
 			}
 
 			Window.Current.Activate();
+			
+			// Integrate with Shell back button behavior on non-Holographic UWP platforms such as Desktop and Mobile
+			if (AnalyticsInfo.VersionInfo.DeviceFamily != "Windows.Holographic")
```

Comment:
Let me see if I can safely access that without needing to do thread hops.  I explicitly went this route since I believe I’ll need to do a DeviceFamliy check for SurfaceHub.  The real way to do this would be some API Contract checks, but this isn't about calling specific APIs but rather having specific behavior with our menu-ing system on these platforms.  Unfortunately all platforms have the AppViewBackButtonVisibility API even if they just no-op on it.